packages/create-app/templates/template-tutorial-todo-app/src/useProjects.ts.hbs

```diff
@@ -0,0 +1,75 @@
+import { OsdkTodoProject } from "{{osdkPackage}}/ontology/objects";
+import { useCallback } from "react";
+import useSWR from "swr";
+import Mocks from "./mocks";
+
+function useProjects() {
+  const { data, isLoading, isValidating, error, mutate } = useSWR<
+    OsdkTodoProject[]
+  >("projects", async () => {
+    // Try to implement this with the Ontology SDK!
+    return Mocks.getProjects();
+    // Solution:
```

Comment:
Do you want to keep these in the code or rely on an external guide to show the solution snippets?